flopy/mt3d/mt.py

```diff
@@ -656,7 +655,11 @@ def load(f, version='mt3dms', exe_name='mt3dms.exe', verbose=False,
             sys.stdout.write('   {:4s} package load...success\n'
                              .format(pck.name[0]))
         ext_unit_dict.pop(btn_key)
-
+        ncomp = mt.btn.ncomp
+        # reserved unit numbers for .ucn, s.ucn, .obs, .mas, .cnf
+        poss_output_units = [range(201, 201+ncomp), range(301, 301+ncomp),

```

Comment:
I'm not sure this is working as intended.  

```
mylist = [range(10), range(10), [17]]
17 in mylist
```
Returns False.

I think what you want here is

```
mylist = list(range(10)) + list(range(10)) + [17]
17 in mylist
```
Returns True.  

So if that line isn't working as intended, I'm wondering if it's needed (since your test passes).  Have I missed something?

Reply:
I see what you're doing now with the any statement.  But I would recommend doing I what I did above, so you can just test

```
if key in poss_output_units:
```

My 2 cents.  Otherwise looks good to me.

Reply:
Thanks @langevin-usgs, good shout, didn't even think about stringing those ranges together like that...updating